submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift

```diff
@@ -1,21 +0,0 @@
-//: Playground - noun: a place where people can play
-
-import Foundation
-
-func fizzbuzz(_ i: Int) -> String {
-    let result = (i % 3, i % 5)
-    switch result {
-        case (0, 0):
-            return "FizzBuzz"
```

Comment:
Your changes still show you deleted files in someone else's submission folder. Please confirm this and fix.